File: C Sharp/Object Validator/Runtime/Null.cs
namespace Quicksilver.ObjectValidator.Runtime;
class Null(bool reversed, string? fieldExpression) : Condition(reversed, fieldExpression)
{
    protected override bool IsFulfilledBy(object? value, string? fullFieldExpression, HashSet<string> passedFields, HashSet<string> failedFields)
    {
        return value == null;
    }
}

File: Java/Object Validator/lib/src/main/java/com/quicksilver/objectvalidator/runtime/Null.java
package com.quicksilver.objectvalidator.runtime;

import java.util.HashSet;

public class Null extends Condition {
    public Null(boolean reversed, String fieldExpression) {
        super(reversed, fieldExpression);
    }

    @Override
    protected boolean isFulfilledBy(Object value, String fullFieldExpression, HashSet<String> passedFields, HashSet<String> failedFields) {
        return value == null;
    }
}
